paddle/fluid/pybind/imperative.cc

```diff
@@ -51,6 +51,8 @@ limitations under the License. */
 namespace paddle {
 namespace pybind {
 
+PyTypeObject *g_VarBase_PyType = NULL;
```

Comment:
Better use either camelcase(variableName) or  underscores(variable_name), not combined.

Reply:
done，thx！